docs/sql/index.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+title: SQL
+---
+
+# SQL Query Language
+
+SQL for Structured Query Language. The SQL query language can be used to create, retrieve and modify data that are stored in the Macrometa geo-distributed fast data platform.
+
+:::note
+If this is your first time with SQL then be sure to check out the [SQL GoT Tutorial](got-tutorial/index.md) before you head off to the in-depth documentation!
+:::
+
+Structured Query Language (SQL) is a standardized programming language that is used to manage relational databases and perform various operations on the data in them.
+Although SQL is an ANSI/ISO standard, there are different versions of the SQL language. Macrometa SQL dialect supports DQL (SELECT statement), DML statements (UPDATE, INSERT and DELETE) and a subset of DDL statements.
+
+The general workflow when executing a query is as follows:
```

Comment:
Love this, but I suggest putting a heading above it. Something like:

## SQL Workflow

Reply:
done done